Activities/GameActivity.cpp

```diff
@@ -1430,55 +1430,50 @@ void GameActivity::Update()
                 m_ControlledActor[player] = 0;
         }
 
-        ///////////////////////////////////////////
         // Player-commanded actor switching
+        if (m_ViewState[player] != ViewState::Observe) {
```

Comment:
This looks like a bit change but it's just adding this if against observe here - this is to stop player controlling actors when forced into observe state